src/main/resources/config/application-inmem.yml

```diff
@@ -0,0 +1,8 @@
+spring:
+  profiles:
+    include: default
+
+security:
+  oauth2:
+    resource:
+      tokenInfoUri: "https://info.services.auth.zalando.com/oauth2/tokeninfo"
```

Comment:
Can we please use example URLs here? Company specific URLs should not appear in this repo, you might set them via environment variable.